lib/client.js

```diff
@@ -212,6 +212,7 @@ Client.prototype._setSequenceArgs = function(argsScheme, args) {
     return args;
   }
   for (var partIndex in argsScheme) {
+    partIndex = partIndex.replace('[]','')
```

Comment:
@AlanMarcell is it possible for you to share the source code of your `kplService.js`? The `client.parseSync()` is "just" a proxy to `wsdl.xmlToObject()` with a `try/catch` block around it. So, if anything, the `xmlToObject()` method would fail.